.github/workflows/verify-pr-commit.yml

```diff
@@ -734,3 +730,89 @@ jobs:
           echo "Actual genesis state: $actual_genesis_state"
           [ $actual_genesis_state = $expected_genesis_state ] || \
             (echo "ERROR: The actual genesis state does not match the expected" && exit 1)
+
+  run-benchmarks:
+    name: Run Benchmarks
+    runs-on: [self-hosted, Linux, X64, benchmark]
+    steps:
+      - name: Check Out Repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{github.event.pull_request.head.sha}}
+      - name: Run Benchmarks?
+        id: run-benchmarks
+        shell: bash
+        run: |
+          commit_message=$(git show -s --format=%s)
+          echo "commit_message: $commit_message"
+          regex='\[run-benchmarks (.*)\]$'
```

Comment:
If I rebase and force push, will it re-trigger benchmarks? It will it the last commit was `add another feature [run-benchmarks all]` (desired), but will it trigger if that commit is `HEAD~2+` (Undesired)?

I think the answer is that it works as desired and this `$commit_message` is only the last commit message.